### PR TITLE
[0.8.0] Nginx latency metrics

### PR DESCRIPTION
**Pull request by igalakhov**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Other (please describe):

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] If it's a backend change, tests for the changes have been added and `go test ./...` runs successfully from the root folder.
- [ ] If it's a frontend change, Prettier has been run
- [ ] Docs have been reviewed and added / updated if needed

## What is the current behavior?

Nginx metrics are in a fairly obscure place

## What is the new behavior?

The plan is to move the 5xx error rate and a new latency histogram to a new metrics tab to the cluster dashboard, and display everything in an organized place there.

### CODE REVIEW

File: dashboard/src/main/home/cluster-dashboard/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -10,20 +10,22 @@ import NodeList from "./NodeList";
...
 }[] = [
   { label: "Nodes", value: "nodes" },
+  { label: "Metrics", value: "metrics" },
   { label: "Namespaces", value: "namespaces" },
   { label: "Settings", value: "settings" },
 ];
 
 export const Dashboard: React.FunctionComponent = () => {
-  const [currentTab, setCurrentTab] = useState<TabEnum>("nodes");
+  const [currentTab, setCurrentTab] = useState<TabEnum>("metrics");
```
Comment (abelanger5) [inline dashboard/src/main/home/cluster-dashboard/dashboard/Dashboard.tsx:28]: Was this just for testing, or do you want to set Metrics as the default tab?

File: dashboard/src/main/home/cluster-dashboard/dashboard/Metrics.tsx [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+import ParentSize from "@visx/responsive/lib/components/ParentSize";
+import AreaChart from "../expanded-chart/metrics/AreaChart";
+import {
+  AvailableMetrics,
+  NormalizedMetricsData,
+} from "../expanded-chart/metrics/types";
+import SelectRow from "../../../../components/form-components/SelectRow";
+import { MetricNormalizer } from "../expanded-chart/metrics/MetricNormalizer";
+
+const resolutions: { [range: string]: string } = {
```
Comment (abelanger5) [inline dashboard/src/main/home/cluster-dashboard/dashboard/Metrics.tsx:18]: These constants should be exported by a shared metrics object, no? I think this is repeated from `MetricsSection`.

File: dashboard/src/main/home/dashboard/ClusterList.tsx [hunk 3]
```diff
@@ -81,6 +81,7 @@ class Templates extends Component<PropsType, StateType> {
         {},
         { project_id: this.context.currentProject.id, cluster_id: clusterId }
       );
+      console.log(res);
```
Comment (abelanger5) [inline dashboard/src/main/home/dashboard/ClusterList.tsx:84]: Remove debug statement

**Review by abelanger5 — CHANGES_REQUESTED** [review]
Seems to be working @igalakhov! A few minor things:
- Need to show units for these metrics -- I believe they're all in seconds
- Rename `Latency Histogram` to `Percentile Response Times`

File: dashboard/src/main/home/cluster-dashboard/dashboard/Dashboard.tsx (hunk 1 above)
Comment (igalakhov) [reply dashboard/src/main/home/cluster-dashboard/dashboard/Dashboard.tsx:28]: Just testing, changed.

**Review by abelanger5 — APPROVED** [review]

**Review by abelanger5 — APPROVED** [review]

**Pull request merged**